typings/flowdock.d.ts

```diff
@@ -6,6 +6,8 @@ declare module 'flowdock' {
 	class Session {
 		constructor(token: string);
 		public get(path: string, data: {}, callback: (error: Error, body: any, result: any) => void): void;
```

Comment:
Probably worth coming up with a type for the callback function here, as it's used in multiple places. In fact, it might be worth doing it for `get`/`post`/`put`, too.